### PR TITLE
crushtool: improve usage/tip messages

### PR DESCRIPTION
**Pull request by xiexingguo**
So they are displayed in a more uniform style and more human-readable.

### CODE REVIEW

**Comment by xiexingguo** [comment]
@dillaman Some cleanups. Could you help me review it? Thanks:-)

File: src/tools/crushtool.cc [hunk 1]
```diff
@@ -128,38 +128,39 @@ void usage()
...
   cout << "\n";
-  cout << "   -i mapfn --add-item id weight name [--loc type name ...]\n";
+  cout << "   -i mapfn --add_item id weight name [--loc type name ...]\n";
   cout << "                         insert an item into the hierarchy at the\n";
   cout << "                         given location\n";
-  cout << "   -i mapfn --update-item id weight name [--loc type name ...]\n";
+  cout << "   -i mapfn --update_item id weight name [--loc type name ...]\n";
   cout << "                         insert or move an item into the hierarchy at the\n";
   cout << "                         given location\n";
-  cout << "   -i mapfn --remove-item name\n"
```
Comment (xiexingguo) [inline src/tools/crushtool.cc]: @dillaman Besides, some of the tips conflict with the implementation. Take "--remove-item" for example, the implementation(check line 415 below here) really considers "--remove_item" as a match, although we can rightfully process both internally.

**Comment by xiexingguo** [comment]
@dillaman @liewegas Revert the underscore changes and only some typos and indention fixes are kept, please review.

**Comment by liewegas** [comment]
I think you ned to run test/run-cli-tests and then move the help.t.err to help.t and commit

**Comment by xiexingguo** [comment]
@liewegas  Fixed. Thank you for your reminder, sir.

**Comment by xiexingguo** [comment]
Below is the test results:
[root@c8 test]# ./run-cli-tests
cli/ceph-authtool/add-key-segv.t: passed
cli/ceph-authtool/add-key.t: passed
cli/ceph-authtool/cap-bin.t: passed
cli/ceph-authtool/cap-invalid.t: passed
cli/ceph-authtool/cap-overwrite.t: passed
cli/ceph-authtool/cap.t: passed
cli/ceph-authtool/create-gen-list-bin.t: passed
cli/ceph-authtool/create-gen-list.t: passed
cli/ceph-authtool/help.t: passed
cli/ceph-authtool/list-empty-bin.t: passed
cli/ceph-authtool/list-empty.t: passed
cli/ceph-authtool/list-nonexistent-bin.t: passed
cli/ceph-authtool/list-nonexistent.t: passed
cli/ceph-authtool/manpage.t: passed
cli/ceph-authtool/simple.t: passed
# Ran 15 tests, 0 skipped, 0 failed.
cli/ceph-conf/env-vs-args.t: passed
cli/ceph-conf/help.t: passed
cli/ceph-conf/invalid-args.t: passed
cli/ceph-conf/manpage.t: passed
cli/ceph-conf/option.t: passed
cli/ceph-conf/sections.t: passed
cli/ceph-conf/show-config.t: passed
cli/ceph-conf/show-config-value.t: passed
cli/ceph-conf/simple.t: passed
# Ran 9 tests, 0 skipped, 0 failed.
cli/crushtool/add-item-in-tree.t: passed
cli/crushtool/add-item.t: passed
cli/crushtool/adjust-item-weight.t: passed
cli/crushtool/bad-mappings.t: passed
cli/crushtool/build.t: passed
cli/crushtool/check-names.empty.t: passed
cli/crushtool/check-names.max-id.t: passed
cli/crushtool/compile-decompile-recompile.t: passed
cli/crushtool/help.t: passed
cli/crushtool/location.t: passed
cli/crushtool/output-csv.t: passed
cli/crushtool/reweight_multiple.t: passed
cli/crushtool/reweight.t: passed
cli/crushtool/set-choose.t: passed
cli/crushtool/show-choose-tries.t: passed
cli/crushtool/straw2.t: passed
cli/crushtool/test-map-bobtail-tunables.t: passed
cli/crushtool/test-map-firefly-tunables.t: passed
cli/crushtool/test-map-firstn-indep.t: passed
cli/crushtool/test-map-indep.t: passed
cli/crushtool/test-map-legacy-tunables.t: passed
cli/crushtool/test-map-tries-vs-retries.t: passed
cli/crushtool/test-map-vary-r-0.t: passed
cli/crushtool/test-map-vary-r-1.t: passed
cli/crushtool/test-map-vary-r-2.t: passed
cli/crushtool/test-map-vary-r-3.t: passed
cli/crushtool/test-map-vary-r-4.t: passed
# Ran 27 tests, 0 skipped, 0 failed.
cli/monmaptool/add-exists.t: passed
cli/monmaptool/add-many.t: passed
cli/monmaptool/clobber.t: passed
cli/monmaptool/create-print.t: passed
cli/monmaptool/create-with-add.t: passed
cli/monmaptool/help.t: passed
cli/monmaptool/print-empty.t: passed
cli/monmaptool/print-nonexistent.t: passed
cli/monmaptool/rm-nonexistent.t: passed
cli/monmaptool/rm.t: passed
cli/monmaptool/simple.t: passed
# Ran 11 tests, 0 skipped, 0 failed.
cli/osdmaptool/clobber.t: passed
cli/osdmaptool/create-print.t: passed
cli/osdmaptool/create-racks.t: passed
cli/osdmaptool/crush.t: passed
cli/osdmaptool/help.t: passed
cli/osdmaptool/missing-argument.t: passed
cli/osdmaptool/pool.t: passed
cli/osdmaptool/print-empty.t: passed
cli/osdmaptool/print-nonexistent.t: passed
cli/osdmaptool/test-map-pgs.t: passed
# Ran 10 tests, 0 skipped, 0 failed.
cli/radosgw-admin/help.t: passed
# Ran 1 tests, 0 skipped, 0 failed.
cli/rbd/help.t: passed
cli/rbd/invalid-snap-usage.t: passed
cli/rbd/not-enough-args.t: passed
# Ran 3 tests, 0 skipped, 0 failed.

**Pull request merged**